Dockerfile.govuk-base

```diff
@@ -6,7 +6,8 @@
 FROM buildpack-deps
 
 # Install chrome and its dependencies
-RUN apt-get update -qq && apt-get install -y libxss1 libappindicator1 libindicator7 --no-install-recommends
+RUN apt-get update -qq && apt-get install -y libxss1 libappindicator1 libindicator7 --no-install-recommends && \
+    apt-get clean && rm -rf /var/lib/apt/lists/*
```

Comment:
Are you sure that `apt-get clean` is necessary given `/etc/apt/apt.conf.d/docker-clean`?

Reply:
:shrug: I just did what the robot told me to do.

Reply:
I think that says everything about the danger to thinking of "linters" as a set of tools.

Reply:
Yeah, I admit I didn't really weigh up pros and cons.